### ISSUE TITLE
Not able to start meeting on iOS but can join a meeting successfully on iOS

### ISSUE THREAD

**Issue by Faizan-Farooq** [evilrat/flutter_zoom_sdk#89]
Hi,
I have been using this plugin for my android app. I am porting the app to iOS and unfortunately the start meeting functionality is not working on iOS. I can successfully join an already started meeting but not start a new meeting.
I checked the example as well and its also not working in the example project provided. I am testing on a real device and not an emulator.
Can anyone guide as to what might be the issue as i am also not getting any logs on xcode.
Thank You!

**Comment by ark-brighthustle** [comment]
> Hi, I have been using this plugin for my android app. I am porting the app to iOS and unfortunately the start meeting functionality is not working on iOS. I can successfully join an already started meeting but not start a new meeting. I checked the example as well and its also not working in the example project provided. I am testing on a real device and not an emulator. Can anyone guide as to what might be the issue as i am also not getting any logs on xcode. Thank You!

any error log

**Comment by Faizan-Farooq** [comment]
> > Hi, I have been using this plugin for my android app. I am porting the app to iOS and unfortunately the start meeting functionality is not working on iOS. I can successfully join an already started meeting but not start a new meeting. I checked the example as well and its also not working in the example project provided. I am testing on a real device and not an emulator. Can anyone guide as to what might be the issue as i am also not getting any logs on xcode. Thank You!
> 
> any error log

yes xcode printed "error was 0" and that too sometimes. it wasn't printed every time

**Comment by Faizan-Farooq** [comment]
@evilrat any follow up on this?
Thanks in advance

**Comment by ark-brighthustle** [comment]
> > > Hi, I have been using this plugin for my android app. I am porting the app to iOS and unfortunately the start meeting functionality is not working on iOS. I can successfully join an already started meeting but not start a new meeting. I checked the example as well and its also not working in the example project provided. I am testing on a real device and not an emulator. Can anyone guide as to what might be the issue as i am also not getting any logs on xcode. Thank You!
> > 
> > 
> > any error log
> 
> yes xcode printed "error was 0" and that too sometimes. it wasn't printed every time

looking into it for solution

**Comment by Faizan-Farooq** [comment]
> > > > Hi, I have been using this plugin for my android app. I am porting the app to iOS and unfortunately the start meeting functionality is not working on iOS. I can successfully join an already started meeting but not start a new meeting. I checked the example as well and its also not working in the example project provided. I am testing on a real device and not an emulator. Can anyone guide as to what might be the issue as i am also not getting any logs on xcode. Thank You!
> > > 
> > > 
> > > any error log
> > 
> > 
> > yes xcode printed "error was 0" and that too sometimes. it wasn't printed every time
> 
> looking into it for solution

Alright. Thank You! I am eagerly waiting for a solution.
And thank you for this awesome package

**Comment by Faizan-Farooq** [comment]
> > > > Hi, I have been using this plugin for my android app. I am porting the app to iOS and unfortunately the start meeting functionality is not working on iOS. I can successfully join an already started meeting but not start a new meeting. I checked the example as well and its also not working in the example project provided. I am testing on a real device and not an emulator. Can anyone guide as to what might be the issue as i am also not getting any logs on xcode. Thank You!
> > > 
> > > 
> > > any error log
> > 
> > 
> > yes xcode printed "error was 0" and that too sometimes. it wasn't printed every time
> 
> looking into it for solution

@evilrat any update on this? Thank You in advance

**Comment by Faizan-Farooq** [comment]
@evilrat sorry to bother again. Any updates on this? I am running past some critical deadlines

**Comment by brandonthomsontz** [comment]
@evilrat are you able to get this package running on iOS simulator?

**Comment by ark-brighthustle** [comment]
> @evilrat are you able to get this package running on iOS simulator?

soon will be update to latest zoom sdk...which might solve the issues

**Comment by Manojdaiya1992** [comment]
@Fanatic6  can you please let me know the configuration for iOS device as I am not able to build the application

**Comment by Faizan-Farooq** [comment]
> @Fanatic6 can you please let me know the configuration for iOS device as I am not able to build the application

iOS builds are not working @Manojdaiya1992

**Comment by Faizan-Farooq** [comment]
@evilrat any update on this or when the new version is coming out?
